geode-core/src/main/java/org/apache/geode/distributed/internal/membership/NetView.java

```diff
@@ -553,6 +554,18 @@ public String toString() {
     // sb.append(ports[i]);
     // }
     sb.append("]");
+    sb.append(", MemberTimeouts : [");
```

Comment:
Unfortunately this level of the system can't use member "names" as keys.  A member is not required to have a "name" in membership services.  Each timeout needs to be associated with an entry in the "members" list, similar to what's done with failure detection ports.